src/data/faq.json

```diff
@@ -2795,17 +2827,19 @@
                                 "anchor": "traffic_monitoring",
                                 "active": false,
                                 "textblock": [
+                                    "<b>Updated ?????r</b>",
                                     "State of the art and well-established encryption mechanisms, for example using HTTP over TLS (HTTPS), ensure that messages are not readable for outside viewers. Metadata is removed before processing user data for the transmission of diagnosis keys. This prevents data from being linked together at database level.",
-                                    "Furthermore, various additional technical measures have been taken. For example, false notifications that will be discarded on the server side are randomly created and sent. These false notifications cannot be distinguished from valid notifications and create a type of background noise. The transmission of TANs and upload of real infection data cannot be distinguished from this background noise. So, even with monitored network traffic, plausible deniability is still possible."
+                                    "Furthermore, various additional technical measures have been taken. For example, false notifications that will be discarded on the server side are randomly created and sent. These false notifications cannot be distinguished from valid notifications and create a type of background noise. The transmission of TANs (As of version 3.0 of the Corona-Warn-App, teleTAN is no longer required to share a positive test result that has not been submitted to the Corona-Warn-App) and upload of real infection data cannot be distinguished from this background noise. So, even with monitored network traffic, plausible deniability is still possible."

```

Comment:
"(As of version 3.0 of the Corona-Warn-App, teleTAN is no longer required to share a positive test result that has not been submitted to the Corona-Warn-App)"

This sentence is in the middle of another sentence. You could leave it there if you change "As" to "as", however that is quite untidy.

I suggest to move it to the end of the paragraph and write:
"Note: As of version 3.0 of the Corona-Warn-App, a teleTAN is no longer used to share a positive test result that has not been submitted to the Corona-Warn-App."

Reply:
Thanks for suggestion, has been adopted.